src/ProjectFileTools/Adornments/HighlightWordTagger.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Operations;
+using Microsoft.VisualStudio.Text.Tagging;
+
+namespace ProjectFileTools.Adornments
+{
+    /// <summary>
+    /// Finds and updates the correct tags for the highlighted text.
+    /// </summary>
+    internal class HighlightWordTagger : ITagger<HighlightWordTag>
+    {
+        ITextView View { get; set; }
```

Comment:
Private data members without logic should be fields
Scope should be explicit (please prefix with `private`)
Private fields (once these are fields) should be `_camelCase` style
If the fields values should not be assignable after the constructor has run, please mark as `readonly`